libs/ui-toolkit/src/components/tabs/tabs.tsx

```diff
@@ -21,7 +21,7 @@ export const Tabs = ({ children, active: activeDefaultId }: TabsProps) => {
     >
       <div className="border-b border-default">
         <TabsPrimitive.List
-          className="flex flex-nowrap overflow-visible"
+          className="flex flex-nowrap overflow-visible bg-neutral-50 dark:bg-neutral-800"
```

Comment:
chore (blocking): I don't agree with these changes, it deviates off brand and looks worse

Reply:
The problem here stems from rendering multiple ag agrids next to each other, which we shouldn't be doing. It creates the additional borders, and if scroll bars appear its even worse. In the second example it looks clean because a) there is only one position open, so there aren't multiple grids in a single tab, and b) the collateral grid is the current one which _is_ still a single grid. As a result it looks much nicer.

If this is a concern lets make sure our designs don't add lots of borders within the tabs where possible

Reply:
Have removed this style for now and created a ticket for David to explore options.
https://github.com/vegaprotocol/ux-and-visual-design/issues/19